spec/integration/multi_xml/xml_spec.rb

```diff
@@ -1,7 +1,13 @@
 require 'spec_helper'
 
 describe Grape::Xml do
-  it 'uses multi_xml' do
-    expect(Grape::Xml).to eq(::MultiXml)
+  if Object.const_defined? :MultiXml
```

Comment:
This change should not be required, the integration spec is specifically for mutli_xml, so it should never be in that `else`.

Reply:
That makes sense about not ever being in the `else` since this is for multi_xml so I'll definitely remove that, but if I don't check for the presence of the constant, this is the only spec that fails for me locally on my ubuntu VM.

Reply:
Run specs with `rake`, I think we cleverly exclude those as needed and run them with their own bundle?

Reply:
Ahh, ok, I was running `bundle exec rspec`  🤦‍♂️